pkg/controller/template/render.go

```diff
@@ -148,6 +148,12 @@ func filterTemplates(toFilter map[string]string, path string, config *RenderConf
 			return nil
 		}
 
+		// Temporarily disable template rendering for all openstack files except "afterburn-hostname.yaml"
+		// This code must be removed when https://github.com/openshift/installer/pull/1959 is merged.
+		if strings.Contains(info.Name(), "openstack") && !strings.Contains(info.Name(), "afterburn-hostname.yaml") {
```

Comment:
should this be more like `config.ControllerConfigSpec.Platform == platformOpenStack && !strings.Contains(info.Name(), "afterburn-hostname.yaml" ` ?

Reply:
@kikisdeliveryservice even better! thanks :)

Reply:
did some debugging on the now failing unit test and my suggestion was way over-inclusive and incorrect. oops? better revert.